dist/faq.html

```diff
@@ -56,8 +56,8 @@
         <div class="question">
             <h4>What is the scoring methodology based on?</h4>
 
-            <p>It is extremely difficult to assign an objective value to a subjective question such as "How bad is not implementing HTTP Strict Transport Security?"  This is complicated by the fact that what may be unnecessary for for one site &mdash; such as implementing Content Security Policy (CSP) &mdash; may be much riskier for a different site.</p>
-            <p>The scores and grades offered by the Mozilla Observatory are based on the opinions of a wide variety of information security professionals, and largely reflect the relative importance written about in the official Mozilla <a href="https://wiki.mozilla.org/Security/Guidelines/Web_Security">web security guidelines</a> and <a href="https://wiki.mozilla.org/Security/Server_Side_TLS">server side TLS guidelines</a>.</p>
+            <p>It is extremely difficult to assign an objective value to a subjective question such as "How bad is not implementing HTTP Strict Transport Security?"  This is complicated by the fact that what may be unnecessary for for one site &mdash; such as implementing Content Security Policy (CSP) &mdash; may be an absolute requirement for a different site.</p>
```

Comment:
There's no site where implementing CSP is an absolute requirement. The reason for this is that CSP is a belt and suspenders mechanism: if you have a site which is insecure without CSP headers, then you have a site which is insecure for a reasonably large class of old browsers.

"Content Security Policy (CSP) is not intended as a first line of defense against content injection vulnerabilities. Instead, CSP is best used as defense-in-depth, to reduce the harm caused by content injection attacks. As a first line of defense against content injection, server operators should validate their input and encode their output."